actions/check_version/index.js

```diff
@@ -0,0 +1,34 @@
+const core = require("@actions/core");
```

Comment:
@Pham-Gia-Huong 
Consider using github.context to get the release tag instead of requesting with api.
const github = require('@actions/github');
const releaseTag = github.context.ref